client/scripts/views/pages/view_snapshot_proposal/index.ts

```diff
@@ -260,41 +305,152 @@ const ViewProposalPage: m.Component<{
-      class: 'ViewProposalPage', 
-      title: 'Snapshot Proposal',
-    }, (!vnode.state.votes && !vnode.state.totals) ? m(Spinner, { active: true }) : [
-      m(ProposalHeader, {
-        snapshotId: vnode.attrs.snapshotId,
-        proposal: vnode.state.proposal,
-      }),
-      m('.PinnedDivider', m('hr')),
-      vnode.state.totals && vnode.state.votes
-      && vnode.state.proposal
-      && m(VotingResults, {
-        choices: vnode.state.proposal.choices,
-        votes: vnode.state.votes,
-        totals: vnode.state.totals,
-        symbol: vnode.state.symbol,
-      }),
-      isActive
-      && author
-      && m(VoteAction, {
-        space: vnode.state.space,
-        proposal: vnode.state.proposal,
-        id: vnode.attrs.identifier,
-        totalScore: vnode.state.totalScore,
-        scores: vnode.state.scores,
-        choices: vnode.state.proposal.choices,
-        votes: vnode.state.votes,
-      })
-    ]);
-  }
+    return m(
+      Sublayout,
+      {
+        class: 'view-snapshot-proposal-page',
+        title: 'Snapshot Proposal',
+      },
+      !vnode.state.votes || !vnode.state.totals || !vnode.state.proposal
+        ? m(Spinner, { fill: true, active: true, size: 'xl' })
+        : [
+            // eslint-disable-next-line no-restricted-globals
+            m('.back-button', { onclick: () => history.back() }, [
+              m('img', {
+                class: 'back-icon',
+                src: '/static/img/arrow-right-black.svg',
+                alt: 'Go Back',
+              }),
+              m('.back-button-text', 'Back'),
+            ]),
+            m(
+              Tabs,
+              {
+                align: 'left',
+                class: 'snapshot-tabs',
+              },
+              [
+                m(TabItem, {
+                  label: 'Proposals',
+                  active: activeTab === 'Proposals',
+                  onclick: () => {
+                    vnode.state.activeTab = 'Proposals';
+                  },
+                }),
+                m(TabItem, {
+                  label: 'Info & Results',
+                  active: activeTab === 'Info & Results',
+                  onclick: () => {
+                    vnode.state.activeTab = 'Info & Results';
+                  },
+                }),
+              ]
+            ),
+            m('.proposal-body', [
+              activeTab !== 'Info & Results' && [
+                m('.proposal-content', [
+                  m(ProposalContent, {
+                    proposal,
+                    votes,
+                    symbol: vnode.state.symbol,
+                  }),
+                ]),
+              ],
+              m('.proposal-info', [
+                m('.proposal-info-box', [
+                  m('.title.padding-x-20', 'Information'),
+                  m('.info-block.padding-x-20', [
+                    m('.labels', [
+                      m('', 'Author'),
+                      m('', 'IPFS'),
```

Comment:
actually, because these labels are paired with data below as rows, we should have this "info block" on the right organized in the following hierarchy: 
```
// Factor out the row into a component that takes in left and right content values
const InfoBlockRow = ....

const { leftContent, rightContent } = vnode.attrs;
returns m(InfoBlockRow, [
   m(label, [leftContent]),
   rightContent,
   ]);

// And then use the component: 
m('.info-block', [
  m(InfoBlockRow, [
      'Author', m(User { ... })]),
  m(InfoBlockRow, [
      'IPFS', m('a', { ... })]),
  m(InfoBlockRow, [
      'VotingSystem', m('.snapshot-type', { ... })]),
....
]);


```

Reply:
Still to be resolved

Reply:
I tried to use both of the suggestions you've given the first time I was doing this. 
The second option can't be used because, in the design, it's a two-column layout and not a flex layout like in snapshot, and to prevent giving a fixed width for the first column, I used all the labels together in a div and the values in another.

![Screenshot 2021-10-07 at 7 41 01 PM](https://user-images.githubusercontent.com/20946466/136401909-d5972166-4814-47e7-9bc9-f92c79509708.png)

![Screenshot 2021-10-07 at 7 43 00 PM](https://user-images.githubusercontent.com/20946466/136402271-3366a0b5-520f-47aa-82a2-14db8923c469.png)
 
The first suggestion makes all the labels inline and not block elements. So I've made it 'paragraph' elements
```
m('.labels', [
    m('p', 'Author'),
    m('p', 'IPFS'),
...
```